dts/bindings/i2c/microchip,xec-i2c.yaml

```diff
@@ -25,3 +25,17 @@ properties:
       type: int
       required: true
       description: Bit position in GIRQ for this device
+
+    sda-gpios:
```

Comment:
This should be `pinctrl-0`, see `dts/bindings/i2c/st,stm32-i2c-v1.yaml` for an example.

Reply:
Ok. I'll study that DT example and make a change once I understand.

Reply:
I need some guidance. Our chip DTSI does not implement a pinctrl with child nodes.  The MEC1501 DTSI has individual nodes directly under soc for each GPIO bank: gpio_000_036, gpio_040_076. These nodes handle GPIO interrupts and I/O.  A pinmux driver handles GPIO pin alternate function programming.  To use the current gpio_xxx_yyy nodes I need the node phandle and the pin bit offset.

Reply:
@galak is pinctrl-0 an agreed naming property name across all I2C drivers? 
@scottwcpg AFAIK the extra device tree property for "pin bit info" can be in microchip,xec-i2c.yaml since it's MCHP specific

Reply:
Let me ask a slightly different question. The MCHP I2C controllers support up to 16 ports. The ports pins are fixed for each chip and some chips have a few ports with 2 sets of SDA/SCL pin to increase pin flexibility.  It is possible with devicetree to create a 16 entry port to pin mapping table with default values allowing the board DTS to override the values.  If a C array could be generated then the driver could easily loop up the pins based only on the port number that we currently have.

Reply:
ping @galak in above question. Is this just a naming convention? Or are there some generic macros that need to be used.
@nashif @dcpleung does @scottwcpg has to follow STM approach or is his way of getting access to pinmux functionality in I2C driver valid?